templates/cooperative.tex

```diff
@@ -45,32 +45,30 @@ \subsection*{\MakeUppercase{Map setup}}
 
 Take the following Map tiles and set them up as shown in the scenario map layout:
 
-\textbf{X × Starting Map tile (I)}
+  \textbf{X × Starting (I) Map Tile}
 \begin{itemize}
     \item ... additional constraints ...
 \end{itemize}
 
-\textbf{Y × Far Map tile (II-III)}
+\textbf{Y × Far Map (II-III) Tile}
 
-\textbf{Z × Near Map tile (IV-V)}
+\textbf{Z × Near Map (IV-V) Tile}
 
 \subsection*{\MakeUppercase{Victory Conditions}}
 ...
 
 \subsection*{\MakeUppercase{Defeat Conditions}}
 ...
 
-\subsection*{\MakeUppercase{Timed events}}
+\subsection*{\MakeUppercase{Additional scenario rules}}
```

Comment:
The idea behing having this at the end was to avoid situation where people would list some rules here and more after "Timed events". "Additional rules" is a very custom section on community scenarios that could be short or crazy long. I think we should only have one section for that and should be the last. Like here:

- https://github.com/qwrtln/Homm3BG-mission-book/pull/19/files#diff-99801d6fd309abf2c66771032615694b975ca01aa46495a516d262261cb597e2R65
- https://github.com/qwrtln/Homm3BG-mission-book/pull/19/files#diff-c1e156b6e832ab5b6865af2c49526a2def5ffcc70541d0985cb1d1084a5985b7R90

And this https://github.com/qwrtln/Homm3BG-mission-book/pull/19/files#diff-c1e156b6e832ab5b6865af2c49526a2def5ffcc70541d0985cb1d1084a5985b7R68 would also go after "Timed events" and before "Enemy movement"

Reply:
Sure, the ordering is now consistent
Victory Conditions
Defeat Conditions
Timed Events
Additional Rules
Everything Else